TaylorFramework/Modules/scissors/ScissorsExtensions.swift

```diff
@@ -72,7 +19,9 @@ protocol StringType {}
 extension String: StringType {}
 
 extension SourceKitRepresentable {
-    var asDictionary: [String: SourceKitRepresentable] { return self as? [String: SourceKitRepresentable] ?? [:] }
+    var dictionaryValue: [String: SourceKitRepresentable] {
+        return self as? [String: SourceKitRepresentable] ?? [:]
```

Comment:
I find returning empty dictionary ambiguous here. It might mean it casted successfully but there are no elements or it might mean it failed to cast, which are completely different situations.